src/variant.rs

```diff
@@ -544,4 +623,42 @@ mod tests {
         let converted = variant.convert_into_cim_type(cim_type).unwrap();
         assert_eq!(converted, Variant::Array(vec![]));
     }
+
+    #[test]
+    fn it_bidirectional_string_convert() {
+        let string = "Test String".to_string();
+        let variant = Variant::from(string.clone());
+        assert_eq!(
+            <Variant as TryInto<String>>::try_into(variant).unwrap(),
```

Comment:
nitpick: you can avoid this by `assert_eq!(variant.try_into().ok(), Some(string.clone()));`